plyer/facades.py

```diff
@@ -82,23 +84,61 @@ def take_video(self, filename, on_complete):
         :type filename: str
         :type on_complete: callable
         '''
+
         self._take_video(filename=filename, on_complete=on_complete)
 
     # private
 
     def _take_picture(self, **kwargs):
         raise NotImplementedError()
-        
+
     def _take_video(self, **kwargs):
         raise NotImplementedError()
 
 
+class Contacts(object):
```

Comment:
Documentation lack of knowing the exact format of the contact list, and usage. The attached example is too complex to understand, but demonstrate it.

Could you explain how a contact is structured? What field are always here, the one that are optional? Best would be to give an output example like:

    >>> from plyer import Contacts
    >>> Contacts.refresh()
    >>> print "I have {} contacts".format(len(Contacts))
    I have 3 contacts
    >>> print Contacts[0]
    { .... }